code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -154,12 +158,15 @@ GLOBAL_LIST_EMPTY(scp173s)
 		if(!(A in our_view))
 			DisableBlinking(A)
 			continue
+		var/mob/living/carbon/human/H = A
 		if(world.time >= next_blinks[A])
```

Comment:
```suggestion
		if(world.time >= next_blinks[H])
```
???

Reply:
Oh, the define for H was previously after that if statement, so it was A. Had to move it for the bitflag, fixing now.